rust/data/presets.json

```diff
@@ -52,46 +54,46 @@
       "canGateGlitch",
       "canOffScreenSuperShot",
       "canGravityJump",
+      "canLateralMidAirMorph",
       "canMockball",
       "canSpringBallJumpMidAir",
-      "canUseFrozenEnemies",
       "canMochtroidIceClimb",
       "canMoonwalk",
       "canMoonfall",
       "canMochtroidIceClip",
       "canCeilingClip",
       "canIframeSpikeJump",
       "canBombHorizontally",
-      "canSnailClimb",
       "canCrumbleJump",
       "canCrystalFlash",
       "canConsecutiveWalljump",
       "canCrossRoomJumpIntoWater",
       "canShinechargeMovement",
       "canMidairShinespark",
-      "canCarefulJump",
-      "canXrayWaitForIFrames",
+      "canXRayWaitForIFrames",
       "canDownBack",
       "canPlayInSand",
       "canHyperGateShot",
       "canSpaceJumpWaterBounce",
       "canWaterShineCharge",
-      "canHeroShot"
+      "canHeroShot",
+      "canMaridiaTubeClip"
     ],
     "notable_strats": [
       "Early Supers Mockball",
       "Alcatraz Escape",
       "Lower Norfair Pillar Room with Power Bombs",
+      "LN Spring Ball Maze Hotarubi Special",
```

Comment:
Hotorubi only uses 26 tiles. Hard requires 27

Reply:
27 with 2 openends 😏 (fix strat to say 2 openends)
The quickcharge part does feel like what you want for hard though, a small one tap.

I'd rather see this notable removed.  But at the same time it is unique.

Reply:
After practicing it a bit, I still think this one should be `VH`. It's about the minimum runway I would expect with `H`, plus a moderately tight jump, plus an air ball, plus pretty obscure.

Although, I do think we should double check the difficulty requirements of different shinespark runways. If this strat is indeed a 26 runway, then I think `H` should be moved from 27 to 26.